spec/features/admin_csv_download_spec.rb

```diff
@@ -21,6 +21,24 @@
     end
   end
 
+  context "User is logged in as a support user" do
+    before do
+      stub_dfe_sign_in_with_role(Admin::AuthController::DFE_SIGN_IN_SUPPORT_ROLE_CODE)
+      visit admin_path
+      click_on "Sign in"
+    end
+
+    scenario "User cannot see a link to download claims" do
+      expect(page).to_not have_content("Download claims")
```

Comment:
Can we test this more robustly? If the copy changed at the same time as the logic to turn it on and off, this would still pass. Can we look for the link by the href instead?